juju-charms/charm-trilio-data-mover/src/README.md

```diff
@@ -0,0 +1,40 @@
+# Overview
+
+TrilioVault Data Mover provides service for TrilioVault Datamover
+on each compute node.
+
+# Usage
+
+TrilioVault Data Mover relies on services from rabbitmq-server.
+Steps to deploy the charm:
+
+juju deploy trilio-data-mover --config <config file>
+juju deploy rabbitmq-server
+juju add-relation trilio-data-mover rabbitmq-server
+
```

Comment:
- juju deploy nova-compute
- juju add-relation trilio-data-mover nova-compute

You missed adding above two lines.